README.md

```diff
@@ -30,7 +34,11 @@ After a game is developed, a common need is to know how the players play, what i
 
 Using the tracker is a really simple proccess. For expanding the Installation guide here you can found some examples of how to setup and use the tracker.
 
-### Tracker access and instatiation
+### Synchronous or asynchronous tracking
+
+C# tracker can work both synchronously and asynchronously, as desired. This behavior can be configured [here](https://github.com/e-ucm/csharp-tracker/blob/master/TrackerAsset/TrackerAsset.cs#L19).
```

Comment:
I would ilustrate this with a comment:

Switch the comment in order to swap between sync and async:
```csharp

#define ASYNC
//#undef ASYNC

```

In this case the tracker is in ASYNC mode.